full-service/src/json_rpc/v1/models/transaction_log.rs

```diff
@@ -159,6 +159,7 @@ impl TransactionLog {
                 txo_id_hex: txo.id.to_string(),
                 recipient_address_id: "".to_string(),
                 value_pmob: txo.value.to_string(),
+                public_key: hex::encode(txo.public_key().unwrap().as_bytes()),
```

Comment:
unwrap here is sketchy, maybe default to an empty string if it fails?

Reply:
Actually here you can just return an error since this already returns a `Result<>`, so that would be better.